packages/grid-manager/src/index.ts

```diff
@@ -211,7 +205,8 @@ class GridManager implements GridManagerProps {
 }
 
 export { GridManager }
-// const machine = new GridManager()
-// machine.start()
-// window.manager = machine
-// export default GridManager
\ No newline at end of file
+
+if(process && process.env === 'development' as any) {
+  window.manager = new GridManager()
```

Comment:
I do not like adding code just for running something in Dev in our library. Why can we just import this in the grid-manager.html? And instantiate the `new GridManager` in that file?

The idea is that we generate different bundles type one that should be compatible with the browser? is that the umd?

Something like this can work?:
```

import('./grid-manager.umd.js').then((Module) => {
    window.manager = new Module.GridManager();
  })
```